EIPS/eip-4400.md

```diff
@@ -183,6 +187,11 @@ contract ERC721Consumable is IERC721Consumable, ERC721 {
 }
 ```
 
+Other implementations of the `Consumable` role:
```

Comment:
Please remove this note to other implementations. EIPs are not an appropriate place to advertise projects.